migration/migrator/data/submitty_db.sql

```diff
@@ -125,7 +125,8 @@ CREATE TABLE users (
     user_email character varying NOT NULL,
     user_updated BOOLEAN NOT NULL DEFAULT FALSE,
     instructor_updated BOOLEAN NOT NULL DEFAULT FALSE,
-    last_updated timestamp(6) with time zone
+    last_updated timestamp(6) with time zone,
+    api_key character varying(255) NOT NULL default encode(gen_random_bytes(16), 'hex')
```

Comment:
As far as I can tell, there is not likely to be collisions since we are going to put this into JWT along with other info like user_id, but yes there can be a UNIQUE.